src/amqp.lisp

```diff
@@ -159,11 +159,13 @@
       (t (error "Unexpected error: ~s" reply-type)))))
 
 (defun verify-rpc-framing-call (state channel result)
-  (when (cffi:null-pointer-p result)
-    (verify-rpc-reply state channel (amqp-get-rpc-reply state))))
+  (if (cffi:null-pointer-p result)
+      (verify-rpc-reply state channel (amqp-get-rpc-reply state))
+      t))
```

Comment:
Why was the return value of `T` added here? This function is only ever called in the unwind handler of an `UNWIND-PROTECT` so the return value should not matter.

Reply:
This is a mistake, I was trying something and forgot to revert it. sorry. reverted